website/app/components/doc/scroll-to-top/index.hbs

```diff
@@ -0,0 +1,11 @@
+{{#if this.isButtonVisible}}
+  <button
+    aria-label="Scroll back to top"
+    class="doc-scroll-to-top"
```

Comment:
we could consider adding a class here, instead of doing the `{{#if this.isButtonVisible}}` above, which shows/hides the button without transition:
```suggestion
    class="doc-scroll-to-top {{if this.isButtonVisible 'doc-scroll-to-top--is-visible'}}"
```
in this was we can set the default opacity to `0`, and transition it to `1` in the `.doc-scroll-to-top--is-visible' CSS selector. A bit more whimsy, what do you think @heatherlarsen ?

Reply:
I think that's a great idea. Let's do it.

Reply:
Added in 000b87a24196a323423000626892b3ae0f8fac99